CHANGELOG.md

```diff
@@ -1,3 +1,126 @@
+### citus v11.1.0 (September 15, 2022) ###
+
+* Adds support for PostgreSQL 15beta4
+
+* Adds a rebalancer that uses background tasks for its execution
+
+* Adds `create_distributed_table_concurrently()` UDF to distribute tables
```

Comment:
I think Marco's blog refers this as `without interrupting the application` instead of `without blocking writes`